pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+It is also super important to mention [Abir Pal](https://twitter.com/imabptweets), who helped to coordinate efforts and was the first point of contact for the community coming from OpenForce channels.
+
+### Google Season of Docs
+
+Success with OpenForce was followed by unexpected announcement that we were accepted as an organization for Google Season of Docs (GSoD) with $10k budget :muscle:
+
+<TwitterTweetEmbed
+  tweetId='1514670329246801926'
+  options={{
+    cards: 'hidden',
+    width: 200,
+    maxWidth: 400
+  }}
+/>
+
+The amazing success of [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and an excellent opportunity for AsyncAPI Initiative to improve documentation big time!
+In mid-May, we will have the names of six interns we want to hire to work on two different projects to have better docs. Alejandra leads a super challenging task to interview as many candidates as possible (around 200!). Keep your fingers crossed.
+
+Alejandra and I will be mentors for interns for the next few months. Stay tuned to watch the progress.
+
+### AsyncAPI Mentorship
+
+Even though the Google Summer of Code (GSoC) application was declined, we did not want to give up. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 
+
+Well tl;dr our TSC just accepted an idea to start our own [AsyncAPI Mentorship](https://github.com/asyncapi/community/discussions/284) program :rocket:
+
+Now it is time until the 15th of May to discuss with mentors [in related ideas](https://github.com/issues?q=is%3Aopen+org%3Aasyncapi+label%3Agsoc) to understand the scope and clearly state you are a candidate for a given idea. Then we will ask TSC to vote and select 10 ideas that AsyncAPI Initiative should sponsor this year.
+
+## AsyncAPI Training
+
+ICYMI, we run an effort to create official AsyncAPI training videos. [Barbaño González](https://www.linkedin.com/in/barbano-gonzalez-moreno/) leads this topic and almost finished [scripts and storyboards](https://github.com/asyncapi/training/pulls) for the first set of videos.
+
+I do not think it is a secret if I say that great content is created only with great input, feedback, and review from people waiting for it. Please join Barbaño's efforts with any help as she is not doing it for herself but for the community.
+
+I recommend you [follow Barbaño's monthly updates](https://gist.github.com/Barbanio), especially if you want to participate.
+
+## Docs Feedback Form
+
+One of our missions is to have kick-ass documentation. Easy right? 
+
+Writing docs is easy, just like writing code. Well, we all know the devil is hidden in details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.
```

Comment:
```suggestion
Writing docs is easy, just like writing code. (Kidding!) Well, we all know the devil is hidden in the details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.
```

Reply:
hmmm, I wasn't `Kidding` in a sense of making a joke. I do believe writing docs and code is easy. But what is complicated is all the work around, figuring out architecture, research, cooperation, etc. Writing final document or JS code is the easiest part in the whole process

Reply:
Interesting.. writing code or documentation is not easy for everyone, so publishing this sentence could come across as minimizing how hard it is for others. I still think you need to leave this as a joke so that it does not come across as discouraging to beginners or people who still struggle ♥️

Reply:
but I can't change my mind, my opinion just because there might be someone that have a different opinion. This is not an article about writing docs or code, not for beginners in this subject. This is an update for the community on things that happened

Reply:
That's fine, then make sure you start saying, **"For me, writing docs is easy, just like writing code."**

It's important to emphasize again this is your opinion because even tho this is a personal blog post, this section still reads like a statement.